ironfish/src/testUtilities/matchers/transaction.ts

```diff
@@ -0,0 +1,19 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { Transaction } from '../../primitives/transaction'
+
+function areTransactionsEqual(a: unknown, b: unknown): boolean | undefined {
```

Comment:
is this used as part of these changes?

Reply:
Yes, implicitly, whenever you call `expect(...).toEqual` on something that has a transaction in it